nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/mongodb/nxserver/config/mongodb-template-directory-config.xml.nxftl

```diff
@@ -2,21 +2,25 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.template.directory.mongodb">
 
-  <require>org.nuxeo.ecm.directories</require>
-  <require>org.nuxeo.template.directory.sql</require>
-  <extension target="org.nuxeo.directory.mongodb.MongoDBDirectoryFactory" point="directories">
+  <#if "${nuxeo.mongodb.directories.enabled}" == "true">
+    <require>org.nuxeo.ecm.directories</require>
+    <require>org.nuxeo.template.directory.sql</require>
+    <extension target="org.nuxeo.directory.mongodb.MongoDBDirectoryFactory" point="directories">
```

Comment:
Usually we don't re-indent stuff when adding FreeMarker `#if` statements so that the final generated file is properly indented. It'll also make the diff simpler to read.